subsys/bluetooth/host/settings.c

```diff
@@ -100,8 +117,41 @@ void bt_settings_encode_key(char *path, size_t path_size, const char *subsys,
 	}
 
 	LOG_DBG("Encoded path %s", path);
+
+	return 0;
 }
-#endif
+
+static int bt_settings_encode_key_no_addr(char *path, size_t path_size, const char *key)
+{
+	size_t len = 3;
+
+	/* Skip if path_size is less than 3; strlen("bt/") */
+	if (len < path_size) {
+		/* Key format:
+		 *  "bt/<key>", "/<key>" is optional
+		 */
+		strcpy(path, "bt/");
```

Comment:
We should consider putting `"bt/"` into a macro or something; it's hardcoded so many places :(

Reply:
Was my first thought too... but did not want to deviate too much from the scope and purpose of the PR.

Reply:
Yeah, that change is better for a different PR IMO